content/millennium/r4/coverages/coverage.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Coverage | R4 API
+---
+
+# Coverage
+
+* TOC
+{:toc}
+
+## Overview
+
+The Coverage Resource provides high-level information about an insurance plan for a specific person or patient. Identifying coverage information would appear on an insurance card, and can be used to pay for health care services. The resource can also be used for "selfpay" where an individual is taking responsibility for the health care services cost, rather than an company or organization.
```

Comment:
nit: `rather than an company or organization` -> `rather than a company or organization`

Reply:
Updated the overview - https://github.com/cerner/fhir.cerner.com/pull/304/commits/88a86bdb579abc932ec0841aff096db0317de3e5#diff-444d12ebae82df3c4c243d0b92726dc2R12